.github/workflows/ci.yml

```
@@ -0,0 +1,77 @@
+on:
+    push: 
+        branches:
+            - main
+            - dev
+    pull_request: 
+        branches:
+            - main
+            - dev
+
+env:
+  CARGO_TERM_COLOR: always
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)